### PR TITLE
Multiple Commits

### PR DESCRIPTION
**Pull request by tsrandrei**
Updated year on _footer. Let's get this to 2019 :)

### CODE REVIEW

File: app/helpers/nav_bar_menu_helper.rb [hunk 1]
```diff
@@ -16,4 +16,8 @@ def count_applicants
     pending_applications = Role.where(state: :screening_completed).count
     "(#{pending_applications})" unless pending_applications.zero?
   end
+  def count_pending_ideas
```
[the remark this answers is not in the document]
Comment (Mehonoshin) [reply app/helpers/nav_bar_menu_helper.rb]: Please add an empty line between methods

File: app/models/application_record.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
-end
+end
```
[the remark this answers is not in the document]
Comment (Mehonoshin) [reply app/models/application_record.rb]: Add a newline please

File: app/views/layouts/shared/_footer.html.haml [hunk 3]
```diff
@@ -7,5 +7,5 @@
         = image_tag 'facebook.png', class: 'facebook'
     %p
       = t('landing.footer.built_by')
-      = succeed ', 2018' do
+      = succeed ', 2019' do
```
Comment (Mehonoshin) [inline app/views/layouts/shared/_footer.html.haml]: Can we use ruby to dynamically find the current year?

File: app/views/web/dashboard/ideas/index.html.haml [hunk 4]
```diff
@@ -8,10 +8,12 @@
       %th= t('dashboard.ideas.table.name')
       %th= t('dashboard.ideas.table.status')
       %th= t('dashboard.ideas.table.author')
+      %th= t('dashboard.ideas.table.created(D/M/Y)')
```
Comment (Mehonoshin) [inline app/views/web/dashboard/ideas/index.html.haml]: this won't work :(

File: db/seeds.rb [hunk 5]
```diff
@@ -33,6 +33,9 @@ def create_user(number)
 # Create members
 5.times do |i|
   user = create_user(i)
+  if i == 1
```
Comment (Mehonoshin) [inline db/seeds.rb]: We can just use `User.first.add_role ...`

File: app/views/layouts/shared/_footer.html.haml (hunk 3 above)
Comment (tsrandrei) [reply app/views/layouts/shared/_footer.html.haml]: I was thinking about that during the coding...So, yes I will switch to ruby also that.

File: app/views/web/dashboard/ideas/index.html.haml (hunk 4 above)
Comment (tsrandrei) [reply app/views/web/dashboard/ideas/index.html.haml]: Why? On my local env is working...

File: db/seeds.rb (hunk 5 above)
Comment (tsrandrei) [reply db/seeds.rb]: I don't know why there is this commit actually. Probably some file that I've touched

**Pull request closed without merging**